src/drivers/driver.ts

```diff
@@ -670,31 +697,17 @@ export abstract class CMakeDriver implements vscode.Disposable {
    */
   private async _beforeConfigureOrBuild(): Promise<boolean> {
     log.debug('Runnnig pre-configure checks and steps');
-    if (this._isBusy) {
-      if (this.ws.config.autoRestartBuild) {
-        log.debug('Stopping current CMake task.');
-        vscode.window.showInformationMessage('Stopping current CMake task and starting new build.');
-        await this.stopCurrentProcess();
-      } else {
-        log.debug('No configuring: We\'re busy.');
-        vscode.window.showErrorMessage('A CMake task is already running. Stop it before trying to configure.');
```

Comment:
Maybe I missed it, but will there be no friendly warning about tasks already running?  It looks like it will just show a generic failure message with a -1 error code now.

Reply:
I moved the message generation without `autoRestartBuild` to `cmake-tools.ts`.